styleguide/STYLEGUIDE.md

```diff
@@ -1,26 +1,31 @@
-## General
+## DuckDuckGo Android Style Guide
 * We care about clean code and aim to make this codebase as self-documenting and readable as possible.
-* We primarily use Kotlin and follow the standard coding conventions outlined in https://kotlinlang.org/docs/reference/coding-conventions.html
+* We primarily use Kotlin and follow coding conventions based on the [Android Kotlin Style Guide](https://developer.android.com/kotlin/style-guide)
 * There may be instances of code that pre-dates our use of this style guide, these can be refactored as we encounter them.
 
-**IDE Setup:**
-* Open Android Studio preferences
-* Tap on code style, select Kotlin
-* Select Set From, Predefined Style, Kotlin Style Guide
-![Setting kotlin style screenshot](IDE_kotlin_style.png)
+###  Android Studio Setup
 
-## Line breaks
-* We use a line margin of 150 rather than the default of 100 as the default causes excessive line breaks. The larger margin allows us to take advantage of modern widescreen monitors with more screen real-estate.
+It is possible to get Android Studio to adhere to these style guidelines, via a rather complex sequence of menus. To make it easier, we've provided a coding style that can be imported into Android Studio.
```

Comment:
The first sentence isn't necessary. How about just chopping it down to:

>We've provided a coding style that can be imported into Android Studio.

Reply:
The reference to "a rather complex sequence of menus" is still here, and still seems unnecessary.

Reply:
I see a 👍on my comment but the line is still in there. 😕